File: app/src/main/java/com/hbjpro/androidvoiceassistant/Interface/OnOrderResult.java
package com.hbjpro.androidvoiceassistant.Interface;

import android.content.Intent;

/**
 * Created by bassa on 08/07/2018.
 */

public interface OnOrderResult {
    void onGetLaunchIntentSuccess(Intent launchIntent);
    void onGetLaunchIntentError();
    //void onSearchInternetSuccess();
    //void onSearchInternetError();
}

File: app/src/main/java/com/hbjpro/androidvoiceassistant/Interface/OnOrderResult.kt
package com.hbjpro.androidvoiceassistant.Interface

import android.content.Intent

interface OnOrderResult {
    fun onGetLaunchIntentSuccess(launchIntent: Intent?)
    fun onGetLaunchIntentError()
    //void onSearchInternetSuccess();
    //void onSearchInternetError();
}
